bonito/openvino/loader.py

```diff
@@ -0,0 +1,99 @@
+# This script provides a method which builds OpenVINO network in runtime
+import numpy as np
+from openvino.inference_engine import IECore, IENetwork
+
+import ngraph as ng
+from ngraph.impl.op import Parameter
+from ngraph.impl import Function, Shape, Type
+
+import torch
+from torch.autograd import Variable
+
+
+nodes = {}
+out = None
+
+def forward_hook(self, inputs, output):
+    global out
+    layer_type = self.__class__.__name__
+
+    params = [value.numpy() for value in self.state_dict().values()]
+
+    inp = nodes[inputs[0].data_ptr()]
+    if layer_type == 'Conv1d':
```

Comment:
preferably we should create nodes from opsets to prevent backward compatibility issues, since nodes created from ng namespace are always taken from the latest opset.

Reply:
We can insert data reshape here instead of later CNNNetwork.reshape call

Reply:
Replaced to `opset4`, thanks!
```python
import ngraph.opset4 as ng
```